flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/internal/TableResultImpl.java

```diff
@@ -248,96 +223,13 @@ public Builder setPrintStyle(PrintStyle printStyle) {
             return this;
         }
 
-        /** Specifies session time zone. */
-        public Builder setSessionTimeZone(ZoneId sessionTimeZone) {
-            Preconditions.checkNotNull(sessionTimeZone, "sessionTimeZone should not be null");
-            this.sessionTimeZone = sessionTimeZone;
-            return this;
-        }
-
         /** Returns a {@link TableResult} instance. */
         public TableResultInternal build() {
+            if (printStyle == null) {
+                printStyle = PrintStyle.rawContent(resultProvider.getRowDataStringConverter());
```

Comment:
nit: usually a builder's `build()` method should not its internal state but just pass the default on to the constructor

Reply:
The default it's the raw content style, but now it needs the `RowDataStringConverter` from `ResultProvider`, that's why I had to move it here.